rcl/test/rcl/test_init.cpp

```diff
@@ -544,6 +544,10 @@ TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_mocked_rcl_init_optio
   auto mock = mocking_utils::inject_on_return("lib:rcl", rmw_init_options_fini, RMW_RET_ERROR);
   EXPECT_EQ(RCL_RET_ERROR, rcl_init_options_fini(&init_options));
   rcl_reset_error();
+  {
+    auto mock = mocking_utils::inject_on_return("lib:rcl", rmw_init_options_fini, RCL_RET_OK);
```

Comment:
```suggestion
    auto mock = mocking_utils::inject_on_return("lib:rcl", rmw_init_options_fini, RMW_RET_OK);
```

Also, why the additional scope?  Same goes for the ones below.

Reply:
Because I didn't want to add a name with `mock1` before.
If using 
`mock = mocking_utils::inject_on_return("lib:rcl", rmw_init_options_fini, RMW_RET_OK);`
, failed as
```
/home/sharedata/Linux/ROS2/docker_ros2_master/src/ros2/rcl/rcl/test/rcl/../mocking_utils/patch.hpp:460:55: error: no match for ‘operator=’ (operand types are ‘mocking_utils::Patch<7, int(rmw_init_options_t*)>’ and ‘mocking_utils::Patch<8, int(rmw_init_options_t*)>’)
```

OK, I'll not use the additional scope.